tag/nitag.yml

```diff
@@ -0,0 +1,1816 @@
+  GetSubscriptionUpdatesResponse:
+    description: List of subscription updates
+    schema:
+      properties:
+        subscriptionUpdates:
+          description: Subscription updates
+          type: array
+          items:
+            $ref: '#/definitions/SubscriptionUpdate'
+        invalidSubscriptionIds:
+          description: List of invalid subscription IDs
+          type: array
+          items:
+            type: string
+
+paths:
+  /v2/subscriptions:
+    post:
+      tags:
+        - subscriptions
+      summary: Create a subscription
+      description: Creates a subscription for a list of tags to get notifications when the tags change.
+      operationId: CreateSubscription
+      parameters:
+        - in: body
+          name: subscription
+          description: Subscription to create
+          schema:
+            $ref: '#/definitions/Subscription'
+          required: false
+      responses:
+        200:
+          description: Success
+          schema:
+            description: Response for creating a subscription
+            type: object
+            title: Create Subscription Response
+            properties:
+              subscriptionId:
+                type: string
+              nonexistentTags:
+                description: List of tags which did not exist
+                type: array
+                items:
+                  type: string
+              alias:
+                type: string
+                description: Name of the subscription
+        400:
+          description: Bad Request
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+    get:
+      tags:
+        - subscriptions
+      summary: Get active subscription IDs
+      description: Returns a list of all active subscription IDs
+      operationId: GetSubscriptions
+      parameters: []
+      responses:
+        200:
+          description: Success
+          schema:
+            description: Get subscriptions response
+            type: object
+            title: Get Subscriptions Response
+            properties:
+              subscriptionIds:
+                description: List of all the active subscription IDs
+                type: array
+                items:
+                  type: string
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+  /v2/subscriptions/{id}:
+    get:
+      tags:
+        - subscriptions
+      summary: Get a subscription
+      description: Returns a subscription's metadata
+      operationId: GetSubscription
+      parameters:
+        - in: path
+          name: id
+          description: Subscription ID
+          type: string
+          required: true
+      responses:
+        200:
+          schema:
+            $ref: '#/definitions/Subscription'
+          description: Success
+        401:
+          $ref: '#/responses/Unauthorized'
+        404:
```

Comment:
Ditto on SL Server returning an error response here. Though I could see it being useful to document 404s? Though I guess it's implied that any URL with an ID in it could return a 404 if the ID doesn't match anything.

Also applies to other routes.

Reply:
I think I'll remove 404s for the same reason that we removed the 400s.